.golangci.yml

```diff
@@ -6,12 +6,8 @@ linters:
     - funlen
```

Comment:
Not necessarily something we need to do for this PR, but I would love to see if we could enable `errcheck` and `scopelint` like we had in CABPK

Reply:
errors were ignored mostly from `init()`, `defer` functions. should i do a  `_ := func()` there?

Reply:
+1 to the use of `_` for the `init` methods, for `defer` it might be good to potentially bubble up the error in a way similar to how we do it for the deferred patch in the cluster controller: https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/cluster_controller.go#L108-L118

Reply:
I've added `errcheck` :D

Reply:
`scopelint` reports error for all table tests :fearful:

Reply:
thanks :D added `scopelint`